runtime/tracing/tracing-core/common/src/aws/smithy/kotlin/runtime/tracing/LoggingTraceProbe.kt

```diff
@@ -14,7 +14,8 @@ import mu.KotlinLogging
 public object LoggingTraceProbe : TraceProbe {
     private fun EventLevel.loggerMethod(): (KLogger, () -> Any?) -> Unit = when (this) {
         EventLevel.Fatal,
-        EventLevel.Error, -> KLogger::error
+        EventLevel.Error,
+        -> KLogger::error
         EventLevel.Warning -> KLogger::warn
```

Comment:
**Comment**: Ugh, don't like the indentation on this one. (Not worth fighting though.)